python/tests/test_wkw.py

```diff
@@ -163,6 +163,36 @@ def test_compress():
             assert np.all(dataset2.read(POSITION, SIZE) == test_data)
 
 
+def test_row_major_order():
+    data_shape = (4, 5, 6)
+    data = generate_test_data(np.uint8, data_shape)
+    print(data)
+    with wkw.Dataset.create("tests/tmp", wkw.Header(np.uint8)) as dataset:
+        dataset.write((0, 0, 0), data)
+        read_data = dataset.read((0, 0, 0), data_shape)
+
+    assert np.all(data == read_data)
+
+    with wkw.Dataset.create("tests/tmp2", wkw.Header(np.uint8)) as dataset:
+        fortran_data = np.asfortranarray(data)
+        dataset.write((0, 0, 0), fortran_data)
+        fortran_read_data = dataset.read((0, 0, 0), data_shape)
+
+    assert np.all(fortran_read_data == read_data)
+    assert np.all(fortran_read_data == fortran_data)
+
+
+def test_row_major_order_with_offset():
+    data_shape = (17, 1, 4)
+    data = generate_test_data(np.uint8, data_shape)
+    print(data)
+    with wkw.Dataset.create("tests/tmp", wkw.Header(np.uint8)) as dataset:
```

Comment:
Did you also test for other dtypes than `np.uint8` and multi-channel data? Maybe add a test case for np.uint16 with two channels or something like that.